compiler/rustc_hir_pretty/src/lib.rs

```diff
@@ -291,8 +295,26 @@ where
     printer.s.eof()
 }
 
+pub fn parsed_attribute_to_string(ann: &dyn PpAnn, attr: &AttributeKind) -> String {
+    to_string(ann, |s| {
+        let this = &mut *s;
+        let style = AttrStyle::Outer;
+        this.print_parsed_attribute_inline(attr, style)
+    })
```

Comment:
leftover from r-a inlining?


```suggestion
    to_string(ann, |s| s.print_parsed_attribute_inline(attr, AttrStyle::Outer))
```